temba/contacts/templatetags/contacts.py

```diff
@@ -199,7 +199,7 @@ def history_class(item):
     if isinstance(item, Msg):
         if item.media and item.media[:6] == 'video:':
             css = '%s %s' % (css, 'video')
-        if item.direction or item.recipient_count:
+        if item.direction or (item.broadcast and item.broadcast.recipient_count > 1):
```

Comment:
Ok well go to know I'm not losing my mind! 

I think `get_activity()` there is using `select_related` for fetching purged broadcasts, but not for messages.